### PR TITLE
Add query strings in get vineyards endpoint

### PR DESCRIPTION
**Pull request by Cody-Price**
1. PR references issue#15

2. Added a search query determination and appropriate happy and sad paths for each for both a name and a region.

3. @saraebbers 

4. I would like to see if we can refactor this into something more dynamic and possibly turn it into a middleware function.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -19,13 +19,42 @@ app.get('/', (request, response) => {
...
+
+  if(Object.keys(queryString).length) {
+    let region;
+    let vineyardName;
+    if(queryString.region) {
+      region = queryString.region.toLowerCase()
+    }
+    if(queryString.name) {
+      vineyardName = queryString.name.toLowerCase()
+      console.log(vineyardName)
```
Comment (saraebbers) [inline server.js]: please remove console.log

File: server.js [hunk 2]
```diff
@@ -56,13 +85,15 @@ app.delete('/api/v1/vineyards/:id', (request, response) => {
...
 });
 
-app.get('/api/v1/vineyards', (request, response) => {
-  // will be used for query
-  //?= query for searching each region, name, 
-  //return 200
-  //return 404
-  //return 500
-});
+// app.get('/api/v1/vineyards', (request, response) => {
```
Comment (saraebbers) [inline server.js]: Lets review need for this to be commented out

File: server.js (hunk 1 above)
Comment (Cody-Price) [reply server.js]: console.log removed

File: server.js (hunk 2 above)
Comment (Cody-Price) [reply server.js]: removed unnecessary commented out code

**Comment by saraebbers** [comment]
great work, thanks

**Pull request merged**